modules/vaos/app/serializers/vaos/appointment_requests_serializer.rb

```diff
@@ -34,25 +43,15 @@ class AppointmentRequestsSerializer
                :provider_id,
                :second_request,
                :second_request_submitted,
-               :patient,
                :best_timeto_call,
-               :appointment_request_detail_code,
                :has_veteran_new_message,
                :has_provider_new_message,
                :provider_seen_appointment_request,
                :requested_phone_call,
                :booked_appt_date_time,
                :type_of_care_id,
                :friendly_location_name,
-               :patient_id,
-               :appointment_request_id,
```

Comment:
We use this variable on the front end, but maybe we shouldn't?

Reply:
you're using appointment_request_id or patient_id? 

patient_id happened to be EDIPI for Judy Morrison. I'm not sure if it is the same for other users or not. 

appointment_request_id I could probably leave in if it makes it easier for you to make requests that way for messages and other things. It's just a duplicate of both id and unique_id. We add id to be consistent with other endpoints -- to me it seemed redundent, but you bring up a good point that it would be necessary for posting messages and stuff.

Reply:
Yeah, appointment_request_id. We could get it elsewhere, but let's leave it in for now, so we don't break the front end. If we want to remove it later, we can update the front end first and then remove it from the back end.